packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/modelparts/ModelSemantics.java

```diff
@@ -20,5 +19,5 @@ public class ModelSemantics implements SupportAdditionalProperties {
 	private List<Object> span;
 
 	@TSOptional
-	private TypingSemantics typing;
+	private Object typing;
```

Comment:
I think we want to keep these `Object` as `JsonNode` just for the fexibility of reading it easy on the back in just in case (it will still translate to `any` for the client)